docker-entrypoint.sh

```diff
@@ -10,7 +10,7 @@ fi
 bundle exec rails zookeeper:upload
 
 # Start sidekiq
-bundle exec sidekiq -d -L kiq.log
+bundle exec sidekiq -d -q ubiquity_json_importer -q default -L kiq.log
```

Comment:
I imagine that `default` is the standard queue holding all the other jobs and `ubiquity_json_importer` is the new queue you introduced for the import, right?

Reply:
Yes and they can be given different priority numbers where sidekiq can check one queue more often than the other and process more item from one queue than the other.

Reply:
Sounds good. Can we make Sidekiq log to STDOUT (so we can get logs inside Kubernete's standard logging)

Reply:
Should be possible but I haven't tried it before. I will google and see.

Reply:
The documentation suggests that by default, the Sidekiq logger logs only to STDOUT
See the line before the

Reply:
@fradeve  The author of sidekiq suggests that we can log to the log file and STDOUT by doing 
  bundle exec sidekiq | tee ./log/sidekiq.log
https://stackoverflow.com/questions/26047790/sidekiq-logging-to-both-terminal-and-log-file

You will notice that he didn't pass the -d option for it to run in detach mode without  passing a log file to using -L, it will fail with you really should set a logfile if you're going to daemonize.

bundle exec sidekiq  -q ubiquity_json_importer -q default  | tee ./log/kiq.log
  instead of this but then it will no longer run in detach mode.
bundle exec sidekiq -d -q ubiquity_json_importer -q default -L kiq.log